third_party/terraform/utils/provider.go.erb

```diff
@@ -317,6 +317,7 @@ end     # products.each do
 				"google_container_node_pool":                   resourceContainerNodePool(),
 				"google_container_registry":                    resourceContainerRegistry(),
 				"google_dataflow_job":                          resourceDataflowJob(),
+				"google_dataflow_flex_template_job":            resourceDataflowFlexTemplateJob(),
```

Comment:
& version guard this entry in the map. There should be similar examples above/below.